app/view/component/MapController.js

```diff
@@ -251,19 +251,21 @@ Ext.define('momo.view.component.MapController', {
 
     setRouting: function(){
         var mapComponentXType = 'momo-component-map';
+        var viewModel =
+            this.getView().up().down(mapComponentXType).getViewModel();
```

Comment:
Yes, i tried it. We need to be more specific here to get the right `viewModel` bounded to the `momo-component-map` component. That's why I use here this pretty complex query.